src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE users(
+    id int GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
```

Comment:
Since it's a learning project it is ok to have `integer` PK, but it's not good practice in general.
Consider changing it to `BigInteger` or `UUID` even we don't have a fat chance to reach this limit